src/chunk.rs

```diff
@@ -0,0 +1,105 @@
+use std::iter::*;
+use std::fmt::Debug;
+
+/// Chunk of code text
+#[derive(Debug)]
+pub enum Chunk {
+    Text(String),
+    EndLine,
+}
```

Comment:
I'd hate to have another Rust formatter here. There are two of them already, both unfinished.